src/main/java/org/jetbrains/bsp/bazel/server/bazel/data/SemanticVersion.java

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.bsp.bazel.server.bazel.data;
+
+public class SemanticVersion implements Comparable<SemanticVersion> {
+
+  private final String version;
+
+  public final String get() {
+    return this.version;
+  }
+
+  public SemanticVersion(String version) {
+    if (version == null) throw new IllegalArgumentException("Version can not be null");
```

Comment:
I think that maybe it would be better to stick with adding brackets around every statement